packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -121,29 +132,62 @@ export class VaButton {
       getButtonText,
       label,
       submit,
+      secondary,
+      outline,
+      big,
+      uswds
     } = this;
 
     const ariaDisabled = disabled ? 'true' : undefined;
     const buttonText = getButtonText();
     const type = submit ? 'submit' : 'button';
 
-    return (
-      <Host>
-        <button
-          aria-disabled={ariaDisabled}
-          aria-label={label}
-          type={type}
-          part="button"
-        >
-          {back && !_continue && (
-            <i aria-hidden="true" class="fa fa-angles-left" />
-          )}
-          {buttonText}
-          {_continue && !back && (
-            <i aria-hidden="true" class="fa fa-angles-right" />
-          )}
-        </button>
-      </Host>
-    );
+    if (uswds) {
+      const buttonClass = classnames({
+        'usa-button': true,
+        'usa-button--secondary': secondary,
+        'usa-button--big': big,
+        'usa-button--outline': back || outline,
+      });
+
```

Comment:
There are three "color variations" that we are not incorporating at this time:

- usa-button--accent-cool
- usa-button--accent-warm
- usa-button--base

![Screenshot 2023-03-10 at 10 42 56 AM](https://user-images.githubusercontent.com/872479/224373174-3eab7af5-de6b-42e9-a133-4474713315c4.png)

When or if we do decide to add them, my recommended approach would be to add a new prop `variation` that would accept one of the three values `accent-cool`, `accent-warm`, or `base`.

And that value can then be used to merge into the `buttonClass` string. For example (with possibly some additional conditional work based on other prop settings):

```
buttonClass += ` usa-button--${variation}`
```

Reply:
I like the idea